Reading List/Controllers/BookController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  BookController.swift
+//  Reading List
+//
+//  Created by Bryan Cress on 5/19/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+
+class BookController {
+    // Read
+    var books: [Book] = []
+    var readbooks: [Book] {
+        books.filter{
+            $0.hasBeenRead == true }
+    }
+    var unreadbooks: [Book] {
+        books.filter{
+            $0.hasBeenRead == false }
+    }
+    
+    // Create
+    func createBook(title: String, reasonToRead: String, hasBeenRead: Bool) {
+        let book = Book(title: title, reasonToRead: reasonToRead, hasBeenRead: hasBeenRead)
+
+        books.append(book)
+        savetoPersistentStore()
+    }
+    
+    //Upload
+    
+    
+    //Delete
+    func deleteBook(book: Book) {
+        // find book passed in as a parameter in book aray
+        guard let index = books.firstIndex(of: book) else { return }
+        
+        //get books array/remove specific book from books aray
+        books.remove(at: index)
```

Comment:
Let's remember to call `savetoPersistentStore()` here so the books that get deleted, stay deleted